content/en/docs/setup/install/multicluster/multi-primary/index.md

```diff
@@ -38,7 +38,7 @@ spec:
     global:
       meshID: mesh1
       multiCluster:
-        clusterName: cluster1
+        clusterName: $CTX_CLUSTER1
```

Comment:
This is not correct. The env variable contains the "context name", which may be different from the "cluster name".

Reply:
Frank is right ... we don't have a variable representing the name of the cluster. It's simply hard-coded to `cluster1`. We could potentially add variables for the cluster names, but then we'd also want to add variables for the mesh and network names for consistency. I'm not sure it's worth it.

Reply:
So the cluster name can be anything and should not be matching the context name? 
If that is the case then I think we don't need variables and hardcodoing works fine.

Reply:
Right, the cluster name is something human-readable, like `cluster1`. The name of the kubernetes context, however, isn't always up to the user ... it could be anything and is often generated by the cloud provider.

Reply:
Thanks @nmittler. I will close this PR.